app/views/events/_form.html.erb

```diff
@@ -15,77 +15,103 @@
       <%= f.text_area :description, id: :event_description, class: 'form-control' %>
     </div>
   </div>
+
   <div class="form-group">
-    <%= f.label :event_type, class: 'control-label col-lg-2' %>
+    <%= f.label :event_type, :class => 'control-label col-lg-2' %>
     <div class="col-lg-10">
-      <%= f.collection_select :type, Event.types, :to_s, :to_s,
+      <%= f.collection_select :type, event_types_enum_localize, :first, :second,
                               {selected: @event.type, include_blank: t('events.new.select_type')},
-                              class: 'form-control', id: 'event_type' %>
+                              :class => 'form-control', :id => 'event_type' %>
```

Comment:
Again this whole file should look a lot cleaner in the way we refactored the model and the views in the process of #145.
You should just be able to create another partial file: `_rankinglist.html.erb` and add your additional fields there. If you open up the link `/events/new?type=rankinglist` it should automatically render the correct form.

Reply:
something that will probably be done in the process of #227

Reply:
The refactoring has already been done and is merged in dev. It's also been done by team swteam in one of their user stories, so #227 is basically obsolete if we just merge it correctly now.